chart/kubeapps/templates/dashboard-config.yaml

```diff
@@ -17,6 +17,12 @@ data:
       gzip_static  on;
 
       location / {
+        # Keep the connection open with the API server even if idle (the default is 60 seconds)
+        # Setting it to 1 hour which should be enough for our current use case of deploying/upgrading apps
+        # If we enable other use-cases in the future we might need to bump this value
+        # More info here https://github.com/kubeapps/kubeapps/issues/766
+        proxy_read_timeout 1h;
```

Comment:
Hmm, I don't fully understand this, isn't this the dashboard nginx server that is only serving static assets for the React application? Why do we need to keep the connection open there? Did you mean to have this configuration in the kubeapps frontend vhost under tiller-proxy instead? https://github.com/kubeapps/kubeapps/blob/512cd5541b692cb1376a60f4310a458945ce1300/chart/kubeapps/templates/kubeapps-frontend-config.yaml#L55

I would consider being more conservative here, e.g. 10 mins instead of 1h, and also only setting this in the tiller-proxy location block and not for the other services.

Also, bear in mind that when the Ingress (e.g. nginx ingress) is used, these settings might not apply. Do we need to add annotations to the Ingress resource?